src/modules/jsonld-author-filter/load.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Filter the author object to add the co-authors.
+ * 
+ * This filter checks if there are co-authors or a single author and
+ * returns a JSON-LD fragment for the author(s).
+ * 
+ * @param array $value {
+ *     @type array $author      The author JSON-LD structure.
+ *     @type int[] $references  An array of post IDs.
+ * }
+ * @param int $post_id The post ID.
+ *
+ * @return string|array A JSON-LD structure.
+ * 
+ * @since 3.51.4
+ * 
+ * @see https://www.geeklab.info/2010/04/wordpress-pass-variables-by-reference-with-apply_filter/
+ */
+function _wl_jsonld_author__author_filter( $args_arr, $post_id ) {
+
+    $author     = $args_arr['author'];
+    $references = $args_arr['references'];
+
+    $coauthor_plugin_path = 'co-authors-plus/co-authors-plus.php';
+
+    // If the co-authors plugin is active.
+    if ( is_plugin_active( $coauthor_plugin_path ) ) {
```

Comment:
To be on the safe side also add `&& function_exists( 'get_coauthors' )`, we don't want to find ourselves breaking a website because this function changed or is renamed.